BattleTank/Input/XInputGamepadTankActionProvider.cs

```diff
@@ -29,16 +29,25 @@ public XInputGamepadTankActionProvider(PlayerIndex padNumber, Buttons speedBoost
         /// <inheritdoc />
         public TankControllerState GetTankControllerState()
         {
-            GamePadState state = GamePad.GetState(PlayerIndex.One);
+            GamePadState state = new GamePadState();
+            if (PadNumber.Equals(PlayerIndex.One))
```

Comment:
A co z ew. inną iloscią graczy? Chodzi mi o to, że jest sztuczne ograniczenie i można by chyba napisać po prostu ``GamePad.GetState(PadNumber)`` i mamy kwestię rozwiązaną, tak? Czy tam coś więcej nie działało?